server.js

```diff
@@ -0,0 +1,50 @@
+import express from "express";
+import axios from "axios";
+import path from "path";
+import { fileURLToPath } from "url";
+
+const app = express();
+const port = 8080;
+
+// Get the current directory
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+// Serve static files from the root directory
+app.use(express.static(path.join(__dirname)));
+
+// Proxy handler
+const proxyHandler = async (req, res) => {
+  try {
+    const targetUrl = `https://tiles.soundscape.services${req.originalUrl}`;
+
+    const response = await axios.get(targetUrl, {
+      headers: {
+        "User-Agent": "YourNewUserAgent",
```

Comment:
Should probably set this to something meaningful.

Reply:
yes... you are right... (I confess that I didn't spend much time on it... and ChatGPT helped ;) )